decodeAttestationObject.ts AttestationStatement type incompatible with early 4.x.x versions of Typescript

## Describe the issue

When using the latest version of @simplewebauthn/server 7.3.0 code fails to compile if using an early 4.x.x version of Typescript. The `get size(): number` getter on the AttestationStatement type causes errrors.

```
node_modules/@simplewebauthn/server/dist/helpers/decodeAttestationObject.d.ts:25:5 - error TS1131: Property or signature expected.

25     get size(): number;
       ~~~

node_modules/@simplewebauthn/server/dist/helpers/decodeAttestationObject.d.ts:25:9 - error TS1005: ';' expected.

25     get size(): number;
           ~~~~

node_modules/@simplewebauthn/server/dist/helpers/decodeAttestationObject.d.ts:25:15 - error TS1005: ';' expected.

25     get size(): number;
                 ~

node_modules/@simplewebauthn/server/dist/helpers/decodeAttestationObject.d.ts:26:1 - error TS1128: Declaration or statement expected.

26 };
   ~
```

## Reproduction Steps

1. Create a new Typescript project using version 4.0.7.
2. npm install the latest version of @simplewebauthn/server.
3. import and use a method from the library.
4. Compile the project.

## Expected behavior

The code will compile and run successfully.

## Potential Solution

After some testing the definition of the size getter could be switch to a simple size property and the implementation (whether getter or property) can be left up to the implementing object.

As an example of how this solves the issue here is a link to a minimal solution. https://www.typescriptlang.org/play?#code/C4TwDgpgBAKhDOwoF4oG8BQVtXgSwC8IAuKAOwFcBbAIwgCcBuDAXwwwGMAbAQ3nlgBGKHipguEKhDLABcROiw4A5hCT4iACgCUpSrQaKcxqPTUV6ZKIObG2bTr36wATCLESpMuQiSZjGtCoAMzMDhJIwMKoZBAA7kI6zBFQwG4x8a5JnAD2ZPA5EgB0XDnKmlFFgdrMHHkFxaXlaVWEEDUYQA

```ts
type Test = {
    size: number;
}

class T1 implements Test {
    get size(): number {
        return 1;
    }
}

class T2 implements Test {
    size = 3;
}

let t1 = new T1();
let t2 = new T2();
console.log(t1.size);
console.log(t2.size);
```

--- a/packages/server/README.md
+++ b/packages/server/README.md
@@ -4,6 +4,7 @@
 ![WebAuthn](https://img.shields.io/badge/WebAuthn-Simplified-blueviolet?style=for-the-badge&logo=WebAuthn)
 [![npm (scoped)](https://img.shields.io/npm/v/@simplewebauthn/server?style=for-the-badge&logo=npm)](https://www.npmjs.com/package/@simplewebauthn/server)
 ![node-lts (scoped)](https://img.shields.io/node/v/@simplewebauthn/server?style=for-the-badge&logo=Node.js)
+![typescript minimum version 4.3](https://img.shields.io/badge/TypeScript-%3E%3D_4.3-3178C6?style=for-the-badge&logo=TypeScript)
 
 - [Installation](#installation)
 - [Usage](#usage)

--- a/packages/server/README.md
+++ b/packages/server/README.md
@@ -4,7 +4,7 @@
 ![WebAuthn](https://img.shields.io/badge/WebAuthn-Simplified-blueviolet?style=for-the-badge&logo=WebAuthn)
 [![npm (scoped)](https://img.shields.io/npm/v/@simplewebauthn/server?style=for-the-badge&logo=npm)](https://www.npmjs.com/package/@simplewebauthn/server)
 ![node-lts (scoped)](https://img.shields.io/node/v/@simplewebauthn/server?style=for-the-badge&logo=Node.js)
-![typescript minimum version 4.3](https://img.shields.io/badge/TypeScript-%3E%3D_4.3-3178C6?style=for-the-badge&logo=TypeScript)
+![typescript minimum version 4.4](https://img.shields.io/badge/TypeScript-%3E%3D_4.4-3178C6?style=for-the-badge&logo=TypeScript)
 
 - [Installation](#installation)
 - [Usage](#usage)

--- a/packages/server/src/helpers/decodeAttestationObject.ts
+++ b/packages/server/src/helpers/decodeAttestationObject.ts
@@ -37,5 +37,5 @@ export type AttestationStatement = {
   get(key: 'certInfo'): Uint8Array | undefined;
   get(key: 'pubArea'): Uint8Array | undefined;
   // `Map` properties
-  get size(): number;
+  readonly size: number;
 };

